pkgs/applications/graphics/meme/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "meme-unstable-${version}";
+  version = "2017-09-10";
+
+  owner = "nomad-software";
+  repo = "meme";
+  goPackagePath = "github.com/${owner}/${repo}";
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+    rev = "a6521f2eecb0aac22937b0013747ed9cb40b81ea";
+    sha256 = "1gbsv1d58ck6mj89q31s5b0ppw51ab76yqgz39jgwqnkidvzdfly";
+  };
+
+  meta = with stdenv.lib; {
+    description = "A command line utility for creating image macro style memes";
+    homepage = "https://github.com/nomad-software/meme";
+    license = licenses.mit;
+    maintainers = [ maintainers.fgaz ];
+    platforms = platforms.linux;
```

Comment:
Upstream says this should work on Mac. Have you tested to see if this builds on mac via Nixpkgs?

Reply:
I don't have a mac to test it with. Should I set it to darwin too and try to trigger a build like you did?

Reply:
I... can't figure out how to do it. `with platforms; [ linux darwin ]` doesn't work. `[ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]` does, but strings are bad, right?